core-infrastructure/src/db/Core.Database/Scripts/051-CreateTrustFinancialTable.sql

```diff
@@ -0,0 +1,144 @@
+IF NOT EXISTS(SELECT *
+              FROM INFORMATION_SCHEMA.TABLES
+              WHERE table_name = 'TrustFinancial')
+    BEGIN
+        CREATE TABLE dbo.TrustFinancial
+        (
+            CompanyNumber                                            nvarchar(8)     NOT NULL,
+            RunType                                                  nvarchar(50)    NOT NULL,
```

Comment:
@jrabbott, is this needed? Will we (or will we need to) actually generate Trust financial information for non-`default` runs? Or worth keeping for the sake of clarity?

Reply:
Worth keeping for clarity. We aren't generating non-default data at the moment, but it could become a thing in future.